napalm_yang/mappings/ios/parsers/config/openconfig-if-ip/ipv4.yaml

```diff
@@ -1,34 +1,35 @@
 ---
 metadata:
-    processor: TextParser
+    processor: TextTree
 
 ipv4:
     _process: unnecessary
     config:
         _process: unnecessary
         enabled:
-            _process: # TODO look at this one, I think this is different depending on IOS version
+            _process:
                 - mode: is_absent
-                  regexp: "(?P<value>^\\s*switchport mode.*$)"
+                  path: "switchport"
         mtu:
             _process: not_implemented
     addresses:
         _process: unnecessary
         address:
             _process:
-                - regexp: "(?P<block>ip address (?P<key>(?P<ip>.*)) (?P<prefix>([0-255]|\\.)*)(?P<secondary> secondary)*)$"
+                - path: "ip.address.?prefix.?mask"
+                  regexp: "^(?P<value>\\d+\\.\\d+\\.\\d+\\.\\d+)"
```

Comment:
But what value is the regex applied against? `prefix` or `mask` part of the path?